variants/AncMed/rules.html

```diff
@@ -1 +1 @@
-Unlike the original variant-rules a fleet occupying the Baleares may <b><u>not</u></b> be used to convoy an army using the normal convoy rules.<br><br>
+Unlike rules of the original variant, a fleet occupying the Baleares may <b><u>not</u></b> be used to convoy an army using the normal convoy rules.<br><br>
```

Comment:
This sounds slightly awkward, can you update that to "Differing from the rules of the original variant, a..."?